jerry-core/jerry-api.h

```diff
@@ -32,15 +33,26 @@ extern "C"
  */
 
 /**
- * Jerry completion codes
+ * Major version of JerryScript API
+ */
+#define JERRY_API_MAJOR_VERSION 1
+
+/**
+ * Minor version of JerryScript API
+ */
+#define JERRY_API_MINOR_VERSION 0
+
+/**
+ * Jerry init flags
  */
 typedef enum
 {
-  JERRY_COMPLETION_CODE_OK                       = 0, /**< successful completion */
-  JERRY_COMPLETION_CODE_UNHANDLED_EXCEPTION      = 1, /**< exception occured and it was not handled */
-  JERRY_COMPLETION_CODE_INVALID_SNAPSHOT_VERSION = 2, /**< snapshot version mismatch */
-  JERRY_COMPLETION_CODE_INVALID_SNAPSHOT_FORMAT  = 3, /**< snapshot format is not valid */
-} jerry_completion_code_t;
+  JERRY_INIT_EMPTY              = (0u),      /**< empty flag set */
```

Comment:
Maybe this would sound better as `JERRY_INIT_DEFAULT`, but I'm okay with `_EMPTY` as well. Up to you.